enterprise/internal/licensing/data.go

```diff
@@ -99,30 +110,33 @@ var planFeatures = map[Plan][]Feature{
 		FeatureACLs,
 		FeatureExplicitPermissionsAPI,
 		FeatureSSO,
+		FeatureBatchChanges{MaxNumBatchChanges: 5}, // 5 is the old unlicensed default
 	},
 	PlanEnterprise0: {
 		FeatureACLs,
 		FeatureExplicitPermissionsAPI,
 		FeatureSSO,
+		FeatureBatchChanges{Unrestricted: true},
```

Comment:
`PlanEnterprise0` did not bundle the `FeatureBatchChanges`, so I think it should be 
```suggestion
		FeatureBatchChanges{MaxNumBatchChanges: 5},
```

?